x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_view/category_stopped_partitions.tsx

```diff
@@ -0,0 +1,118 @@
+  const [tableRow, setTableRow] = useState<Array<{ count?: number; example: string }>>([]);
+  const [hasStoppedPartitions, setHasStoppedPartitions] = useState(false);
+
+  let _resultsSubscription;
+
+  const columns = useMemo(
+    () => [
+      {
+        field: 'partitionName',
+        name: i18n.translate(
+          'xpack.ml.newJob.wizard.pickFieldsStep.stoppedPartitionsPreviewColumnName',
+          {
+            defaultMessage: 'Stopped partition name',
+          }
+        ),
+        render: (partition: any) => (
+          <EuiText size="s">
+            <code>{partition}</code>
+          </EuiText>
+        ),
+      },
+    ],
+    []
+  );
+  function setResultsWrapper(results: Results) {
+    loadCategoryStoppedPartitions();
+  }
+
+  async function loadCategoryStoppedPartitions() {
+    const results = await ml.results.getCategoryStoppedPartitions([jobCreator.jobId]);
+
+    if (
+      results?.jobs !== undefined &&
+      Array.isArray(results?.jobs[jobCreator.jobId]) &&
+      results.jobs[jobCreator.jobId].length > 0
+    ) {
+      const stoppedPartitionsPreview = results.jobs[jobCreator.jobId];
+      // once we have reached number of stopped partitions we wanted to show as preview
+      // no need to keep fetching anymore
+      if (
+        stoppedPartitionsPreview.length >= NUMBER_OF_PREVIEW &&
+        _resultsSubscription !== undefined
+      ) {
+        _resultsSubscription.unsubscribe();
+        _resultsSubscription = undefined;
+      }
+      setHasStoppedPartitions(true);
+      setTableRow(
+        stoppedPartitionsPreview.slice(0, NUMBER_OF_PREVIEW).map((partitionName) => ({
+          partitionName,
+        }))
+      );
+    }
+  }
+
+  useEffect(() => {
+    // only need to run this check if jobCreator.perPartitionStopOnWarn is turned on
+    if (jobCreator.perPartitionCategorization && jobCreator.perPartitionStopOnWarn) {
+      // subscribe to result updates
+      _resultsSubscription = resultsLoader.subscribeToResults(setResultsWrapper);
+      return () => {
+        if (_resultsSubscription) {
+          _resultsSubscription.unsubscribe();
+        }
+      };
+    }
+  }, []);
+
+  return (
+    <>
+      {hasStoppedPartitions && (
+        <>
+          <EuiSpacer />
+          <div>
+            <FormattedMessage
+              id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationStoppedPartitionsTitle"
+              defaultMessage="Stopped partitions"
+            />
+          </div>
+
+          <EuiCallOut
+            color={'warning'}
+            size={'s'}
+            title={
+              <FormattedMessage
+                id="xpack.ml.newJob.wizard.pickFieldsStep.stoppedPartitionsExistCallout"
+                defaultMessage="There may be fewer results than there could have been because stop_on_warn is turned on. Both categorization and subsequent anomaly detection have stopped for some partitions for job '{jobId}' where the categorization status has changed to 'warn'."
```

Comment:
I think it is fine to go with the option of showing a preview of 5 stopped partitions maximum, and to stop making the calls when it has reached that limit.

I wonder if this message needs an edit for use in the wizard. Maybe something like

```
Both categorization and subsequent anomaly detection have stopped for some partitions for job '{jobId}' as categorization was not producing good results and stop_on_warn was set to true. Therefore there may be fewer results for this job than if stop_on_warn was set to false.
```

@lcawl any suggestions?

Reply:
It seems like what we want is a brief explained of why there are "stopped partitions" or what that means. To that end, I suggest something like this:

> Per-partition categorization and stop_on_warn settings are enabled. Some partitions in job '{jobId}' are unsuitable for categorization and therefore are not included in the categorization or anomaly detection analysis.

If that's not the intention of this message, however, please let me know and I'll keep pondering.

Reply:
Updated the message to @lcawl's version here: https://github.com/elastic/kibana/pull/75709/commits/618fa8fab2d16c4406f007e3d8c367b7c1ad5795

Reply:
One question on the suggested text - do you know if the categorization status goes to `warn` if that means the job will not give any results for that partition at all, or if it means that we will stop generating new results for that partition? The new text makes it sound like the first case i.e. we won't get any results at all for that partition.

Reply:
OK, so it looks like the job will stop generating new results for the stopped partition, so there may still be some results for the stopped partitions before it went to `warn`. So I'd suggest an edit to the text from @lcawl to
```
Per-partition categorization and stop_on_warn settings are enabled. Some partitions in job '{jobId}' are unsuitable for categorization and have been excluded from further categorization or anomaly detection analysis.
```

Reply:
That's a great clarification, thanks!

Reply:
Updated here https://github.com/elastic/kibana/pull/75709/commits/676dfff60738922e0ffdb7fa6c192fcc588c9a0d